app.py

```diff
@@ -75,3 +80,14 @@ def post_like():
     song = flask.request.get_json()
     like = firebase.like_song(id_token, song)
     return flask.jsonify(like)
+
+
+@app.route('/_unlike', methods=['DELETE'])
```

Comment:
FYI - you can simplify this routing a bit; routes are unique on the combination of url/method, not just url.  Given that, you could do something like:

```
@app.route('/_like/<id>', methods=['DELETE']) # Delete a like
@app.route('/_like', methods=['POST']) # Create a new like
@app.route('/_like', methods=['GET']) # Get a list of likes
```

Typical CRUD is:
```
@app.route('/things', methods=['GET']) # Get a list of things
@app.route('/things', methods=['POST']) # Create a thing
@app.route('/things/<id>', methods=['GET']) # Get a specific thing
@app.route('/things/<id>', methods=['PATCH']) # Update a specific thing
@app.route('/things/<id>', methods=['DELETE']) # Delete a specific thing
```